token/core/zkatdlog/nogh/driver/base.go

```diff
@@ -81,12 +84,37 @@ func (d *base) newWalletService(
 		deserializerManager,
 		ignoreRemote,
 	)
+	// owner role
+	// we have one key manager for fabtoken and one for each idemix issuer public key
+	var kmps []common2.KeyManagerProvider
```

Comment:
since you know the exact size its better to initialize it with the length of pp.IdemixIssuerPublicKeys. This makes it a bit more clear that they are 1-1 mapped.

Reply:
okay